jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -73,10 +110,80 @@ def to_numpy_dtype(dt):
       self.assertEqual(to_numpy_dtype(jtu._dtype(x)),
                        to_numpy_dtype(jtu._dtype(y)))
 
+  def _enable_limitations_collection(self):
+    # No need to monkey patch again
+    if self._collect_limitations:
```

Comment:
This test will not work if we have patched already but have not collected any limitations, right?

Reply:
Hm, I think it will not work at all since I should set `self._collect_limitations` to True below! I fixed it in a new commit, good catch!